### ISSUE TITLE
AssertionError when adding timer as simple valve

### ISSUE THREAD

**Issue by lureder** [valiquette/homebridge-orbit-irrigation#84]
When checking the box to show timer as simple valve, I get the following error saying that displayName can not be empty. My accessory has a name in the B-Hyve app.

```
AssertionError [ERR_ASSERTION]: Accessories must be created with a non-empty displayName.
    at new Accessory (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:491:11)
    at new PlatformAccessory (/usr/local/lib/node_modules/homebridge/src/platformAccessory.ts:71:9)
    at valve.createValveAccessory (/usr/local/lib/node_modules/homebridge-orbit-irrigation/devices/valve.js:23:24)
    at /usr/local/lib/node_modules/homebridge-orbit-irrigation/orbitplatform.js:187:38
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
```

**Comment by lureder** [comment]
Also receiving this warning in the log if that's helpful when I uncheck the "display as a simple valve" checkbox

```
[18/03/2024, 16:19:59] [homebridge-orbit-irrigation] This plugin generated a warning from the characteristic 'Name': characteristic value expected string and received undefined. See https://homebridge.io/w/JtMGR for more info.
[18/03/2024, 16:19:59] [homebridge-orbit-irrigation] This plugin generated a warning from the characteristic 'Configured Name': characteristic value expected string and received undefined. See https://homebridge.io/w/JtMGR for more info.
[18/03/2024, 16:19:59] [homebridge-orbit-irrigation] This plugin generated a warning from the characteristic 'Model': characteristic value expected string and received undefined. See https://homebridge.io/w/JtMGR for more info.
```

**Comment by valiquette** [comment]
Is this just an issue when using the option for a simple valve?

If there is any question with the name on the orbit side, pleas check at https://techsupport.orbitbhyve.com

**Comment by lureder** [comment]
Yes, only get this error when checking the simple valve option. Double checked the names on the link above and everything is in Orbit side.

**Comment by valiquette** [comment]
Thanks, I will look into it

**Comment by lureder** [comment]
I think I figured out what was wrong…

By default, the name of the zone is the same as the name the accessory. But when I tried to edit the zone name, the field was blank. I then gave it a different name than the what I named the accessory and tried the “simple valve” option again and this time it worked right away with the name I gave the zone in the app rather than the name of the accessory.

Hope that made a sense! :)
